### ISSUE TITLE
I can not show some emoji on Android which sent from iOS 

### ISSUE THREAD

**Issue by chihung93** [googlefonts/noto-emoji#339]
🥲🥲🥲🥲🥲🥲🥲🥲🥲🥲🥲🥲
As far as I can tell that emoji available on iOS, but it's not included in Noto-emoji.

🥲  = Smiling Face with Tear
Ref: https://emojipedia.org/smiling-face-with-tear/

Thanks.
<img width="112" alt="Screen Shot 2021-04-15 at 16 01 00" src="https://user-images.githubusercontent.com/5199109/114843239-d0cb3180-9e03-11eb-9f17-7fa9a4e8053a.png">

**Comment by RadekBledowski** [comment]
It's included and you are using Samsung emojis not Noto emoji

**Comment by chihung93** [comment]
@RadekBledowski  I have downloaded a sample of the font on Google Github Example, but It's not showing on my Samsung phone.

**Comment by zachliu** [comment]
i'm using it on my Linux Mint 20.2 (Ubuntu 20.4), but
in addition to :smiling_face_with_tear: i also don't see:

:face_exhaling: 
:face_in_clouds: 
:face_with_spiral_eyes: 

also, 

:rock:

**Comment by pappasam** [comment]
🥲

**Comment by jbicha** [comment]
@zachliu This GitHub project is for Google's Noto Emoji. Google doesn't control what versions of Noto Emoji that Ubuntu or Linux Mint include.

**Comment by zachliu** [comment]
@jbicha but i just did `git pull` and `time make -j` and installed it on my machine :thinking:

**Comment by jbicha** [comment]
@zachliu Maybe you didn't install it correctly. Your issue is not the same as the original issue reported here.

**Comment by dym-sh** [comment]
entire @github is affected by this: https://github.community/t/github-website-ui-missing-404-some-emojis-in-the-status-picker/193470/5

**Comment by zachliu** [comment]
@dym-sh Exactly! These seem to be some brand new emojis introduced 2020-2021.  Now i can confirm that the latest `NotoColorEmoji.ttf` does have them. But the funny thing is the system can display them at some places (text editor, or even GitHub comment editor) but not all places (GitHub published comments).

Text Editor - right click - `Insert Emoji`
![2021-09-01_18-02](https://user-images.githubusercontent.com/14293802/131872971-47b64009-b74b-4074-ba31-d9ad462474f4.png)

GitHub comment editor (pasted from the Text Editor)
![2021-09-02_11-29](https://user-images.githubusercontent.com/14293802/131872987-c2474f9b-493f-4aee-8edd-e6601ed24752.png)

**Comment by dym-sh** [comment]
`Insert Emoji` list is system-level, will prob be updated with the next point-release anyways

**Comment by zachliu** [comment]
do you know how linux systems index/search these emojis? how exactly is the name "rock" mapped to 🪨?  i'm just curious
